homeassistant/components/media_player/harman_kardon_avr.py

```diff
@@ -0,0 +1,127 @@
+"""
+Support for interface with an Harman/Kardon or JBL AVR.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/media_player.harman_kardon_avr/
+"""
+import logging
+import voluptuous as vol
```

Comment:
Please keep he imports grouped.

Reply:
Sorry, I'm not sure what you mean by this. I have based this platform on the [`denonavr`](https://github.com/home-assistant/home-assistant/blob/dev/homeassistant/components/media_player/denonavr.py)

Reply:
There need to be a [blank line](https://www.python.org/dev/peps/pep-0008/#imports) between `import logging` and `import voluptuous as vol`.